parlai/agents/transformer/modules/__init__.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
```

Comment:
what about `functions`?

Reply:
Good catch! There actually is another agent that uses `create_position_codes` (I initially thought the functions weren't used anywhere else).